packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimePickerContent.tsx

```diff
@@ -222,8 +222,8 @@ const EmptyRecentList = memo(() => {
       <Trans i18nKey="time-picker.content.empty-recent-list">
         <div>
           <span>
-            It looks like you haven&apos;t used this time picker before. As soon as you enter some time intervals,
-            recently used intervals will appear here.
+            It looks like you haven't used this time picker before. As soon as you enter some time intervals, recently
```

Comment:
This will not pass the lintinng rules as we use `react/no-unescaped-entities`. Do we need to change it?

Reply:
@ifrost, Thank you for your feedback. 
The reason I remove `&apos;` and instead put `'` is because in the UI the entity `&apos;` is printing and it is not rendered to `'`.

Somehow when this comes from the `grafana.json` as a string, it is not rendered properly. 
![image](https://github.com/grafana/grafana/assets/20130692/c53115b4-92e7-4058-ab7a-f7a9077405d6)


Can we remove `&apos;` from the JSON file and keep it as it is in this file?

Reply:
Good catch!

> Can we remove &apos; from the JSON file and keep it as it is in this file?

If we do that, the build won't pass because we use `react/no-unescaped-entities` which prevents using characters like `'` inside JSX tags.

There are a few options to address this:

(1) Keep `'` but disable linting for that specific line.
(2) Use `&apos;` and unescape it with `<Trans i18nKey="time-picker.content.empty-recent-list" shouldUnescape={true}>`
(3) Keep `'` and [adjust linting rules](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md#forbid) to exclude `"` and `'` which may show up in translations frequently.
(4) Keep `'` but generate it outside of JSX tag with `t(...)`
(5) Change the copy to _"You have not (...)"_ :D 

For this PR probably (1) would be okay, but I wonder if we should also think about (3) or find some other way to simply use `'` in translation tags without any need to have HTML entities in translation files / unescape / skip linting manually. By excluding `'` from the linting rule we could keep translation files more readable without any extra handling. @grafana/grafana-frontend-platform what do you reckon?

Reply:
Option 4 of generating it with `t` sounds good for now and then we don't have to disable any sort of linting, but yeah we understand it's not ideal to have to change how things are coded.

Reply:
Thanks @JoaoSilvaGrafana! Do you think it's worth adding it to guidelines?

@aditipatelpro please try to go with option (4) for now.

Reply:
Thank you for your feedback @ifrost, @JoaoSilvaGrafana, I have implemented it.
Could you please review it?

Reply:
Looks good with `t(...)` function used for the translations! 🙌  I think it'd be good to make key names a bit more meaningful instead of using numbers as suffixes, e.g.:

`time-picker.content.empty-recent-list-text1` -> `time-picker.content.empty-recent-list-info`
`time-picker.content.empty-recent-list-text2` -> `time-picker.content.empty-recent-list-docs`

@JoaoSilvaGrafana wdyt?

Reply:
Ha yeah I just replied to a different comment about this further down, but these names look good too 🙌

Reply:
👀 That makes sense  @ifrost, I have made the changes. Could you please review it @ifrost , @JoaoSilvaGrafana?